backend/tests/Feature/ButtonVisitorsApiTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+
+namespace Tests\Feature;
+
+use App\Entities\User;
+use App\Entities\Visitor;
+use App\Entities\Website;
+use DateTime;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class ButtonVisitorsApiTest extends TestCase
+{
+    use RefreshDatabase;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->user = factory(User::class)->create();
+    }
+
+    public function testButtonCountVisitors()
+    {
+        factory(Website::class)->create();
+        $firstDate = new DateTime('@1565560845');
```

Comment:
use instead readable dates, and transform to timestamps where it is necessary. It will be easier to support